airbyte-integrations/bases/base-python/docs/concepts/batches.md

```diff
@@ -0,0 +1,26 @@
+# Intermediate Concepts
```

Comment:
not sure exactly where this will fit in the docs but decided to start writing docs as I go and will stitch it all together with #2761